rdt/transformers/pii/anonymizer.py

```diff
@@ -0,0 +1,138 @@
+        locales (list):
+            List of localized providers to use instead of the global provider.
+        missing_value_replacement (object or None):
+            Indicate what to do with the null values. If an integer or float is given,
+            replace them with the given value. If the strings ``'mean'`` or ``'mode'`` are
+            given, replace them with the corresponding aggregation. If ``None`` is given,
+            do not replace them. Defaults to ``None``.
+        model_missing_values (bool):
+            Whether to create a new column to indicate which values were null or not. The column
+            will be created only if there are null values. If ``True``, create the new column if
+            there are null values. If ``False``, do not create the new column even if there
+            are null values. Defaults to ``False``.
+    """
+
+    DETERMINISTIC_TRANSFORM = False
+    DETERMINISTIC_REVERSE = False
+    INPUT_TYPE = 'pii'
+    null_transformer = None
+
+    @staticmethod
+    def check_provider_function(provider_name, function_name):
+        """Check that the provider and the function exist.
+
+        Attempt to get the provider from ``faker.providers`` and then get the ``function``
+        from the provider object. If one of them fails, it will raise an ``AttributeError``.
+
+        Raises:
+            ``AttributeError`` if the provider or the function is not found.
+        """
+        module = getattr(faker.providers, provider_name)
+        if provider_name == 'BaseProvider':
+            getattr(module, function_name)
+
+        else:
+            provider = getattr(module, 'Provider')
+            getattr(provider, function_name)
+
+    def __init__(self, provider_name=None, function_name='lexify', function_kwargs=None,
+                 locales=None, missing_value_replacement=None, model_missing_values=False):
+        self.data_length = None
+        self.provider_name = provider_name if provider_name else 'BaseProvider'
+        self.function_name = function_name
+        self.function_kwargs = deepcopy(function_kwargs) if function_kwargs else {}
+        self.check_provider_function(self.provider_name, self.function_name)
+
+        self.missing_value_replacement = missing_value_replacement
+        self.model_missing_values = model_missing_values
+
+        self.locales = locales
+        self.faker = faker.Faker(locales)
+        self._function = getattr(self.faker, function_name)
+
+    def get_output_types(self):
+        """Return the output types supported by the transformer.
+
+        Returns:
+            dict:
+                Mapping from the transformed column names to supported data types.
+        """
+        output_types = {}
+        if self.null_transformer and self.null_transformer.models_missing_values():
+            output_types['is_null'] = 'float'
+
+        return self._add_prefix(output_types)
+
+    def _fit(self, columns_data):
```

Comment:
`columns_data` should just be `data` (we changed it in the other transformers as well since this is always a series). 

Also, add the documentation for fit/transform/reverse_transform (just copy paste from another transformer).

Reply:
👍🏻